### DISCUSSION TITLE
Problem importing from /config/pyscript/modules

### DISCUSSION THREAD

**Discussion by dorbitbrown** [custom-components/pyscript#208] [Q&A]
My script got too large to load reliably, so I've refactored it into modules which I've saved in /config/pyscript/modules.  For example:
/config/pyscript/modules/physics.py.  

But now when my script in /config/pyscript/scripts runs and tries to import from one of those modules, it errors out.  For example:
....ERROR (MainThread) [custom_components.pyscript.scripts.mkwd] Exception in </config/pyscript/scripts/mkwd.py> line 23:
....from physics import sun_meridian_bearing, desired_cover_position
....^
....ModuleNotFoundError: No module named 'physics'

I discovered that /config/pyscript/modules was not in sys.path, so I appended it, but that didn't change anything.

Any suggestions?

**Comment by craigbarratt** [comment]
Can you post a few lines from the top of `/config/pyscript/modules/physics.py`?

**Reply by dorbitbrown** [reply]
I must sheepishly admit it was entirely user error!  Too embarrassed to even say what it was.  

I apologize for taking up your time.

Thanks again for developing Pyscript!

**Reply by craigbarratt** [reply]
Glad it's resolved!  Also, minor comment - pyscript doesn't use sys.path for searching for user modules.
